### PR TITLE
OSSM-4872: Modify upgrade content

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OSSM-4872

Link to docs preview: https://65351--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/upgrading-ossm#ossm-upgrade-known-issues_ossm-upgrade

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by alexissolanas** [comment]
LGTM!

**Comment by pbajjuri20** [comment]
LGTM..!

**Review by pbajjuri20 — APPROVED** [review]

File: modules/ossm-upgrade-known-issues.adoc [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 Known issues that may affect your upgrade include:
 
+* When upgrading an Operator, custom configurations for Jaeger or Kiali may be reverted. Before upgrading an Operator, note any custom configuration settings for the Jaeger or Kiali objects in the {SMProductShortName} production deployment so that you can recreate them.
```
Comment (sabrinajess) [inline modules/ossm-upgrade-known-issues.adoc:10]: Replace "may be" with "might be". 
Per ISG "To indicate possibility, use 'might'. To indicate permission, use 'may'."

**Review by sabrinajess — COMMENTED** [review]
One small nit, otherwise LGTM. Tagging @ousleyp to confirm.

File: modules/ossm-upgrade-known-issues.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-upgrade-known-issues.adoc]: Thanks for reviewing @sabrinajess. I've updated the PR.

**Pull request merged**